build_tools/cirrus/arm_wheel.yml

```diff
@@ -19,7 +19,9 @@ macos_arm64_wheel_task:
     # Only the latest Python version is built and tested on CirrusCI, the other
     # macos arm64 builds are on GitHub Actions
     - env:
-        CIBW_BUILD: cp311-macosx_arm64
+        CIBW_BUILD: cp312-macosx_arm64
+        # TODO: remove when Python 3.12 is released
+        CIBW_PRERELEASE_PYTHONS: True
```

Comment:
This can now be removed, since cibuildwheel [v2.15.0](https://github.com/pypa/cibuildwheel/releases/tag/v2.15.0) CPython 3.12 wheels are built by default.

Reply:
Thanks, good to know!

I'd rather wait for Python 3.12 to be released before removing all of them because we rely on `CI_BW_PRERELEASE_PYTHONS` on Windows for building the custom Docker image.